state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -2772,8 +2772,14 @@ pub mod pallet {
 			)
 			.and_then(|amount| C::ChainAmount::try_from(amount).ok())
 			.unwrap_or_else(|| {
-				log::warn!("Unable to calculate input amount required for gas of {required_gas:?} for input asset ${input_asset:?}. Estimating the input amount based on a reference price.");
-				C::input_asset_amount_using_reference_gas_asset_price(input_asset,required_gas)
+				log::warn!("Unable to calculate input amount required for gas of {required_gas:?} for input asset ${input_asset:?}. Estimating the input amount based on either oracle price or reference price.");
```

Comment:
I think we should remove this warning, especially now that we have a reliable fallback option. We don't need this polluting the node logs.